WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -117,7 +117,7 @@ import WordPressFlux
         didSet {
             if tagSlug != nil {
                 // Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/5223
-                title = tagSlug
+                title = NSLocalizedString("TOPIC", comment: "Topic page title")
```

Comment:
Do we really want "TOPIC" here? It seems that the title is screaming at me. 😳

Reply:
Lol was just following the designs
![Screen Shot 2020-07-27 at 1 51 26 PM](https://user-images.githubusercontent.com/1335657/88590873-47bf6080-d010-11ea-8128-c06f3ccb440a.png)
but since the titles are not caps anyways, I'll update

Reply:
Done in #14525